src/main/webapp/style.css

```diff
@@ -307,3 +307,46 @@ input[type="color"] {
   padding: 8px;
   height: 18px;
 }
+
```

Comment:
We may want to start dividing this document up into sections using big comments as headers; we've got a lot of different groups of css going on here now.

Reply:
Done!